ntbs-service/Program.cs

```diff
@@ -1,152 +1,837 @@
 ﻿using System;
```

Comment:
There's an awful lot of code in this file - I think it might be worth splitting it up into different files. Perhaps one for each of the Configure methods?